vital/algo/resection_camera.cxx

```diff
@@ -0,0 +1,86 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/**
+ * \file
+ * \brief resection_camera instantiation
+ */
+
+#include <vital/algo/algorithm.txx>
+#include <vital/algo/resection_camera.h>
+
+/// \cond DoxygenSuppress
+INSTANTIATE_ALGORITHM_DEF( kwiver::vital::algo::resection_camera );
+/// \endcond
+
+using namespace std;
+
+namespace kwiver {
+
+namespace vital {
+
+namespace algo {
+
+resection_camera
+::resection_camera()
+{
+  attach_logger( "algo.resection_camera" );
+}
+
+camera_perspective_sptr
+resection_camera
+::resection( frame_id_t frmID,
```

Comment:
```suggestion
::resection( frame_id_t frame,
```
...or `frame_id`, but in any case, please choose something more in line with prevailing naming conventions.